catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/CameraBrickTest.java

```diff
@@ -0,0 +1,89 @@
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.bricks.CameraBrick;
+import org.catrobat.catroid.content.bricks.ChooseCameraBrick;
+import org.catrobat.catroid.ui.ScriptActivity;
+import org.catrobat.catroid.uiespresso.util.BaseActivityInstrumentationRule;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.checkIfBrickAtPositionShowsString;
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.checkIfSpinnerOnBrickAtPositionShowsString;
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.clickSelectCheckSpinnerValueOnBrick;
+
+public class CameraBrickTest {
+	private int cameraBrickPosition;
+	private int chooseCameraBrickPosition;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		cameraBrickPosition = 1;
+		chooseCameraBrickPosition = 2;
+		Script script = BrickTestUtils.createProjectAndGetStartScript("cameraBrickTest");
+		script.addBrick(new CameraBrick());
+		script.addBrick(new ChooseCameraBrick());
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Test
+	public void testCameraBrick() {
+		checkIfBrickAtPositionShowsString(cameraBrickPosition, R.string.brick_video);
+		checkIfSpinnerOnBrickAtPositionShowsString(R.id.brick_video_spinner, cameraBrickPosition, R.string
+				.video_brick_camera_on);
+		checkIfBrickAtPositionShowsString(chooseCameraBrickPosition, R.string.brick_choose_camera);
+		checkIfSpinnerOnBrickAtPositionShowsString(R.id.brick_choose_camera_spinner, chooseCameraBrickPosition, R.string
+				.choose_camera_front);
+
+		clickSelectCheckSpinnerValueOnBrick(R.id.brick_choose_camera_spinner,
```

Comment:
please use checkIfValuesAvailableInSpinnerOnBrick(List<Integer> stringResourceIdValues, int spinnerResourceId, int brickPosition) instead of clicking on every available value